### PR TITLE
Update releasenotes.md

### PR DESCRIPTION
**Pull request by czubair**
test

### CODE REVIEW

File: docs/policies/releasenotes.md [hunk 1]
```diff
@@ -30,11 +30,12 @@ The release notes consist of four sections:
 
 We don't want to advertise every single bug fix as most of them do not impact the way customers think about developing the client.  The change log provides an exhaustive list of changes.  We don't need to duplicate it.
 
-A critical change is one that the developer would either need to know or want to know.
+However, in the release notes we do want to list critical changes for customers. A critical change is one that the developer would either need to know or want to know. Use the following section headers (Security Fixes, Breaking Changes, New Featues, and Key Bug Fixes) for the defined critical changes:
```
Comment (adrianhall) [inline docs/policies/releasenotes.md:33]: Featues => Features, also, highlight the headings - eg. back-ticks or italics.

File: docs/policies/releasenotes.md [hunk 2]
```diff
@@ -48,6 +49,19 @@ You can either leave the library out of the release notes, or add a note such as
 
 The release manager will produce the point-in-time snapshot of the versions and ensure the release notes use the snapshot to generate the quick links tables.
 
-## Who publishes the release notes?
+## Who publishes the release notes and when are they published?
+
+* For all on-time releases, the release manager will merge and publish the release notes in the current month's release folder. 
+* For all out-of-band releases, the release notes in the following month.
```
Comment (adrianhall) [inline docs/policies/releasenotes.md:55]: For all out-of-band releases, the release notes will be published in the following on-time release-notes.

File: docs/policies/releasenotes.md [hunk 3]
```diff
@@ -48,6 +49,19 @@ You can either leave the library out of the release notes, or add a note such as
...
+* For all on-time releases, the release manager will merge and publish the release notes in the current month's release folder. 
+* For all out-of-band releases, the release notes in the following month. 
+
+Release notes will locked for GA and preview libraries after the blog publication date (1 week following release date). (Note: all changelogs should be done on release day)
+
+## What do I need to do for an out-of-band release?
+
+The release manager will merge and publish the release notes in the following month's release folder.
+
+* GA libraries must contact Adrian Hall with their readme.md and changelog.md link so he can tweet about the release. You may optionally create a page with this information and send it to Adrian.
```
Comment (adrianhall) [inline docs/policies/releasenotes.md:63]: I don't think we want to name specific people.   

* You may optionally provide social media outreach for out-of-band releases.  Contact the _Community Engagement Manager_ for details on this at least 7 working days prior to the release.

File: docs/policies/releasenotes.md (hunk 1 above)
Comment (czubair) [reply docs/policies/releasenotes.md]: Okay done

**Pull request merged**